### ISSUE TITLE
Missing error text details for values of enumeration when enum type is JSON object

### ISSUE THREAD

**Issue by thjaeckle** [networknt/json-schema-validator#1094]
Hello.

I have a JsonSchema with an enumeration of JSON objects:
```json
{
    "title": "Severity",
    "type": "object",
    "properties": {
        "name": {
            "title": "Name",
            "description": "The human readable name of the severity",
            "type": "string"
        },
        "cardinality": {
            "title": "Cardinality",
            "description": "The severities cardinality, the higher the worse it gets",
            "type": "integer",
            "minimum": 0,
            "maximum": 50,
            "multipleOf": 10
        }
    },
    "additionalProperties": false,
    "required": [
        "name",
        "cardinality"
    ],
    "enum": [
        {
            "name": "EMPTY",
            "cardinality": 0
        },
        {
            "name": "OK",
            "cardinality": 20
        },
        {
            "name": "UNKNOWN",
            "cardinality": 30
        },
        {
            "name": "WARNING",
            "cardinality": 40
        },
        {
            "name": "CRITICAL",
            "cardinality": 50
        }
    ],
    "default": {
        "name": "UNKNOWN",
        "cardinality": 30
    }
}
```

When I now validate the following JSON against that:
```json
{
   "name": "FOO",
   "cardinality": 50
}
```

I get a validation error, but with the following error:
```
enum -> does not have a value in the enumeration [, , , , ]
```

Seems like only scalar JSON values are contained in the generated error text, could that be?

**Comment by justin-tay** [comment]
Thanks for the reporting the issue!

**Comment by stevehu** [comment]
Thanks @thjaeckle for reporting the bug and @justin-tay for fixing it. I have released 1.5.1 with the fix in.
